### DISCUSSION TITLE
Is iOS 16.1 required?

### DISCUSSION THREAD

**Discussion by huong-i3** [reactiveui/ReactiveUI#3678] [Q&A]
I get this warning

`
Warning CA1416: This call site is reachable on: 'iOS' 14.0 and later, 'maccatalyst' 14.0 and later. 'ReactiveViewController' is only supported on: 'iOS' 16.1 and later, 'maccatalyst' 16.1 and later. (CA1416)
`

Our app needs min supported iOS version as 14.0. How to get rid of this warning?

**Comment by glennawatson** [comment]
You should be able to set your sdk version to 16.1 (iOS submission requirement). The minimum version can be much earlier though.

**Reply by huong-i3** [reply]
@glennawatson 
Sorry do you mean that I should set the SupportedOSPlatformVersion in csproj file to 16.1?
If so, it seems like SupportedOSPlatformVersion has to be consistent with MinimumOSVersion plist value ([link](https://github.com/xamarin/xamarin-macios/issues/12336))
